cfy_manager/config.py

```diff
@@ -113,9 +122,26 @@ def dump_config(self):
                 )
             )
 
-    def load_config(self):
+    def load_config(self, config_files=None):
         self._load_defaults_config()
-        self._load_user_config()
+        for config_file in config_files or [CONFIG_FILE_NAME]:
+            config_file_path = self._sanitized_config_path(config_file)
+            if config_file_path:
+                logger.info('Loading configuration from '
+                            '{0}'.format(config_file_path))
+                self._load_user_config(config_file_path)
+            else:
+                logger.warning(
+                    'Configuration file path {0} is not valid. Use a file '
+                    'located in {1} directory'.format(config_file_path,
+                                                      CLOUDIFY_HOME_DIR)
+                )
+
+    def _sanitized_config_path(self, file_path):
+        """Returns a file path in the CLOUDIFY_HOME_DIR or None."""
+        sanitized = abspath(join(CLOUDIFY_HOME_DIR,
+                                 re.sub(r'^(\.*/)+', '', file_path)))
+        return sanitized if sanitized.startswith(CLOUDIFY_HOME_DIR) else None
```

Comment:
why?! I want to `docker run -v /xxx.yaml:/tmp/xxx.yaml`, requiring a specific location goes very much against the idea of being able to provide paths..?

Reply:
It's about improving security.  Making it harder for users to accidentally loose files they might be needing one day.